go.mod

```diff
@@ -4,19 +4,15 @@ go 1.13
 
 require (
 	github.com/alecthomas/kong v0.2.11
+	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
+	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d // indirect
```

Comment:
Do you know what introduced these?

Reply:
iirc they appeared when the dependency on `oam-kubernetes-runtime` was removed.

Reply:
Hmm that is somewhat surprising to me :thinking: I guess if they don't flag `make check-diff` then they are accurate though...